docs/test-tools/test-backend-service.md

```diff
@@ -1,8 +1,8 @@
 # Test deployed service
 
-1. Testing with manually generated Azure Active Directory Token
+1. Testing manually with an Azure Active Directory Token
 
-    - [Generate Azure Active Directory token manually](../test-tools/generate-aad-token-manually.md) to call secure Apis of Azure Communication Services Authentication Hero sample.
+    - [Generate Azure Active Directory token (Refer to steps 1 - 5)](../../MinimalClient/README.md) to call secure Apis of Azure Communication Services Authentication Hero sample.
```

Comment:
so there will be no more ref to generate-aad-token-manually.md doc?

Reply:
Can we add : "Additionally you could also use a [Manual way of generating AAD Token](link). Note: This is a longer process "